domain_source/domain_source.module

```diff
@@ -23,6 +23,9 @@ const DOMAIN_SOURCE_FIELD = 'field_domain_source';
  * @param string $bundle
  *   The bundle being created.
  *
+ * @throws \Drupal\Core\Entity\EntityStorageException
```

Comment:
We aren't explicitly throwing these exceptions, so I don't know that we would document them here. Are there cases where Exceptions are thrown? Under what conditions?

Reply:
The execeptions are thrown by functions called by this function, specifically by entityTypeManager's getStorage() and setComponent(), (IIRC not on work system at present).

My question about 'policy around documenting exceptions' relates to exactly this.

 - if the exceptions are documented, then docblock-aware editors can see that it is possible to receive those exceptions from a call even though there is no explicit throw;
 - if the exceptions are caught and dealt with, then we're good;
 - if neither, a caller can be surprised by a thrown exception.

Happy to follow your lead.

Reply:
So would it be better to remove the @throws for the moment and deal with the issue as a whole at a later time?